src/localization/locale/en.ts

```diff
@@ -86,6 +86,8 @@ export const en = {
     updateWorkQueue: 'Work queue updated',
   },
   info: {
+    artifact: 'Artifact',
+    noResults: 'No tracked results',
```

Comment:
Curious why we sometimes refer to artifacts as "results". "Tracked" is also a little confusing to me. Are there untracked results?